### PR TITLE
Vtcca 2731 fix

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: test/controllers/propertyLinking/FileUploadControllerSpec.scala [hunk 1]
```diff
@@ -58,16 +57,6 @@ class FileUploadControllerSpec extends VoaPropertyLinkingSpec with FakeObjects{
       status(result) mustBe OK
     }
 
-  it should "return remove file success" in {
```
Comment (DanielConnelly) [inline test/controllers/propertyLinking/FileUploadControllerSpec.scala]: why are we removing this test?

Comment (habeeb01) [reply test/controllers/propertyLinking/FileUploadControllerSpec.scala]: same comment

Comment (habeeb01) [reply test/controllers/propertyLinking/FileUploadControllerSpec.scala]: I can see now that that test has been replaced

**Comment by habeeb01** [comment]
LGTM 👍

**Comment by DanielConnelly** [comment]
LGTM 👍

**Pull request merged**